chain/pool/txpool.go

```diff
@@ -602,7 +602,7 @@ func (tp *TxnPool) CleanBlockValidationState(txns []*transaction.Transaction) er
 			for txnHash, err := range errMap {
 				log.Errorf("RefreshBlockValidationState error for txn %x: %v", txnHash, err)
 			}
-			invalidTxns := make([]*transaction.Transaction, len(errMap))
+			invalidTxns := make([]*transaction.Transaction, 0, len(errMap))
```

Comment:
How to reproduce this panic?

Reply:
When a txn returned error in L600 but cannot be found in txns (so invalidTxns contains nil). It's very hard to reproduce. I only saw it once.